torch/csrc/jit/codegen/cuda/transform_view.cpp

```diff
@@ -195,18 +197,22 @@ class SplitTransform final : public ViewTransform {
     Val* remainder = ceilDiv(id->extent(), factor);
 
     // outer loop IterDomain
-    IterDomain* factor_id = new IterDomain(
-        new Int(0), factor, id->getParallelType(), id->getIterType(), true);
+    IterDomain* factor_id = IrBuilder::create<IterDomain>(
+        IrBuilder::create<Int>(0),
+        factor,
+        id->getParallelType(),
+        id->getIterType(),
+        true);
 
     // inner loop IterDomain
-    IterDomain* remainder_id = new IterDomain(
-        new Int(0),
+    IterDomain* remainder_id = IrBuilder::create<IterDomain>(
+        IrBuilder::create<Int>(0),
```

Comment:
Agreed, but I was thinking to put it in the container, not in the builder, since we create new builder instances frequently that end up for the same container.